apps/dashboard/src/routes/lms/community/ask/+page.svelte

```diff
@@ -10,23 +11,37 @@
   import TextEditor from '$lib/components/TextEditor/index.svelte';
   import TextField from '$lib/components/Form/TextField.svelte';
   import { profile } from '$lib/utils/store/user';
+  import { fetchCourses } from '$lib/components/Courses/api';
 
   let errors: {
     title?: string;
+    courseId?: string;
   } = {};
   let fields = {
     title: '',
-    body: ''
+    body: '',
+    course_id: ''
```

Comment:
@tunny17 No, don't change this to snake case because of the validation, you should have just changed the validation to `courseId`, instead of the fields and the UI code to snake case

Reply:
i had to change to fields.course_id from fields.courseId becasue the name of the column on on the supabase table is "course_id", it was a bug but i can rename the column on the table to courseId if  you want me to

Reply:
no keep the column that way, but map it back to camel case when you get it from the backend